sonar-plugin-api/src/main/java/org/sonar/api/utils/text/JsonWriter.java

```diff
@@ -48,17 +49,24 @@
  *     .endObject()
  *     .close();
  * </pre>
+ * </p>
```

Comment:
my bad, javadoc on nulls is missing. Could you please add "<p>By default, null values are not serialized. Use {@link #setSerializeNulls(true)} to enable writing of null values.</p>" ? 
Thanks